ODIN_II/SRC/partial_map.cpp

```diff
@@ -29,6 +29,7 @@
 #include "odin_globals.h"
 
 #include "netlist_utils.h"
+#include "netlist_cleanup.h"
```

Comment:
This seems like a very important file, and should have a file-scope comment saying what kind of code belongs here, and a pointer to the document that talks about partial mapping (both with Odin-II and Yosys+OdinII).  Or else what partial mapping is should be commented here, if there isn't any separate document talking about it.

Reply:
I have added a comprehensive file-scope comment to this file, hope it looks better now